docs/index.js

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable no-alert */
 /* eslint-disable no-plusplus */
 /* eslint-disable no-unused-vars */
+
 const humburgerBtn = document.querySelector('.humburger');
 const navMenu = document.querySelector('.nav-menue');
 const Portfolio = document.querySelectorAll('.card');
```

Comment:
- [ ] You did a great job on the styles but for this project, you need to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=bLYVUGwB5b3Vd3OO-0) template as much as possible.  Please use the blur effect in the mobile-menu background and use the correct color for background `#6070FF`, and make sure it matches the Figma design
  - You can use this property for blur `filter: blur(px);`.
### Your design

![image](https://user-images.githubusercontent.com/83803414/209121361-0dd17a8c-6b32-4d47-83ba-e7be8749f13c.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/209121330-a5e5eaa2-b419-44ce-9bf7-d7663ccb3d77.png)